components/04-views/dashboard/wishlist/wishlist.hbs

```diff
@@ -0,0 +1,29 @@
+{{ render '@header' }}
+
+<div class="dashboard container">
+    <div class="grid">
+        <div class="grid__column grid__column--1-4 dashboard__sidebar">
+            {{ render '@dashboard-nav' }}
+        </div>
+
+        <div class="grid__column grid__column--3-4 dashboard__content">
+            <div class="grid">
+                {{#each wishlistItems}}
+                    <div class="dashboard__item grid__column grid__column--1-2">
+                        {{> '@product--with-add-to-cart-button' item }}
+
+                        {{> '@input--textarea' textarea }}
+
+                        {{> '@input--inline' qty }}
+
+                        {{> '@dashboard-actions' itemAction }}
+                    </div>
+                {{/each}}
+
+                {{> '@dashboard-actions' actions }}
+            </div>
+        </div>
+    </div>
+</div>
+
+{{ render '@footer' }}
```

Comment:
- Too small space between columns grid
- Too wide qty input

Reply:
changed

Reply:
buttons 100% width for mobile.
IMO, we should make it everywhere, to not see smth like this:
<img width="446" alt="screen shot 2017-09-07 at 12 20 02" src="https://user-images.githubusercontent.com/15080214/30158866-ea21fdbc-93c6-11e7-8332-3d3688a237be.png">

Reply:
changed

Reply:
👍